LibraBFT/Impl/Consensus/RoundManager.agda

```diff
@@ -35,9 +39,33 @@ module LibraBFT.Impl.Consensus.RoundManager where
 processCommitM : LedgerInfoWithSignatures → LBFT (List ExecutedBlock)
 processCommitM finalityProof = pure []
 
--- IMPL-TODO: implement this
+------------------------------------------------------------------------------
+
+generateProposalM
+  : Instant → NewRoundEvent
+  → LBFT (Either ErrLog ProposalMsg)
+
 processNewRoundEventM : Instant → NewRoundEvent → LBFT Unit
-processNewRoundEventM now nre = pure unit
+processNewRoundEventM now nre@(NewRoundEvent∙new r _) = do
+  logInfo fakeInfo   -- (InfoNewRoundEvent nre)
```

Comment:
This seems to be missing a `logEE`

Reply:
Mark mentioned he noticed that some functions do not include logEE and he was OK with that.
So I have been leaving them out.